lib/adagrams.rb

```diff
@@ -0,0 +1,125 @@
+def draw_letters
+  letter_distribution = {
+    "a" => 9,
+    "b" => 2,
+    "c" => 2,
+    "d" => 4,
+    "e" => 12,
+    "f" => 2,
+    "g" => 3,
+    "h" => 2,
+    "i" => 9,
+    "j" => 1,
+    "k" => 1,
+    "l" => 4,
+    "m" => 2,
+    "n" => 6,
+    "o" => 8,
+    "p" => 2,
+    "q" => 1,
+    "r" => 6,
+    "s" => 4,
+    "t" => 6,
+    "u" => 4,
+    "v" => 2,
+    "w" => 2,
+    "x" => 1,
+    "y" => 2,
+    "z" => 1,
+  }
+  letter_pool = []
+  letter_distribution.each do |str, n|
+    n.times do
+      letter_pool << str.downcase
```

Comment:
When you compute something like this in a method you should think about whether or not it should be a constant.  Here it's perfectly reasonable to write the method this way but it's worth keeping in mind.

If it's expensive to compute (like, if it needs to read from a file) then you may want to store it once instead of generating it multiple times.